markhor_bringup/launch/markhor_common.launch

```diff
@@ -1,4 +1,8 @@
 <launch>
+    <param name="robot_description" command="xacro '$(find markhor_description)/urdf/markhor.urdf.xacro'" />
+    <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher" respawn="false" output="screen">
+        <remap from="/joint_states" to="$(arg robot_namespace)/joint_states" />
```

Comment:
The file doesn't have an arg robot_namespace. Also in this current state my visualization does not work properly. Adding this solves the issue.
![image](https://user-images.githubusercontent.com/46896242/214124380-daef1e18-e9dd-4908-a359-a0b038d28354.png)

Reply:
Ok so after a few tests, I think the robot_state_publisher should be in flippers.launch so that it can be in the same namespace as the flippers. Otherwise the flipper movements aren't showing the visualization. The robot_description param would also need to be moved back to flippers.launch

Reply:
robot_description brought back to flippers.launch, joint state publisher uses it. For the state publisher, I disagree, it is robot wide so not specific to the tracks, you need it to view the lidar data, lidar should not depend on flippers.